### ISSUE TITLE
Python 3 compatibility

### ISSUE THREAD

**Issue by elad661** [google/gtfs-realtime-bindings#17]
It seems that the Python bindings are not compatible with Python 3.

```
Python 3.4.3 (default, Jun 29 2015, 12:16:01) 
[GCC 5.1.1 20150618 (Red Hat 5.1.1-4)] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> from google.transit import gtfs_realtime_pb2
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/lib/python3.4/site-packages/google/transit/gtfs_realtime_pb2.py", line 22, in <module>
    from google.protobuf import reflection as _reflection
  File "/usr/lib/python3.4/site-packages/google/protobuf/reflection.py", line 68, in <module>
    from google.protobuf.internal import python_message
  File "/usr/lib/python3.4/site-packages/google/protobuf/internal/python_message.py", line 848
    except struct.error, e:
                       ^
SyntaxError: invalid syntax
```

**Comment by elad661** [comment]
Oops, I guess that's a generic issue with protobuf itself. I'll close this issue and find the right place to file it.

**Comment by elad661** [comment]
Oh, with the beta version of protobuf (which supports Python 3), the bindings fail with a different error

```
>>> from google.transit import gtfs_realtime_pb2
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/lib/python3.4/site-packages/google/transit/gtfs_realtime_pb2.py", line 366, in <module>
    has_default_value=False, default_value=unicode("", "utf-8"),
NameError: name 'unicode' is not defined
```
so... reopening the issue. sorry for the noise.

**Comment by jbb352** [comment]
Having the same issue.  Any news on a release for Python3?

**Comment by araichev** [comment]
Having the same issue.

**Comment by robbiet480** [comment]
I have a working branch! It's available [here](https://github.com/robbiet480/gtfs-realtime-bindings/tree/python3). Just needed to follow instructions in python/UPGRADING.md and change the protobuf version to at least 3.0.0b3.

**Comment by robbiet480** [comment]
Fixed in PR #20!
